src/DotNetWorker.Core/Context/Features/IFunctionInputBindingFeature.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Functions.Worker.Context.Features
+{
+    /// <summary>
+    /// Provides a mechanism to bind function inputs.
+    /// </summary>
+    public interface IFunctionInputBindingFeature
```

Comment:
Renamed [IModelBindingFeature](https://github.com/Azure/azure-functions-dotnet-worker/blob/storage-extension-5.1.0-preview1/src/DotNetWorker.Core/Context/Features/IModelBindingFeature.cs#L8) and exposing as public type.